<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualWidgetFrm</class>
 <widget class="QWidget" name="ManualWidgetFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Read the manual</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPdfManual">
        <property name="text">
         <string>PDF manual</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelHtmlManual">
        <property name="text">
         <string>HTML manual</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Download the manual</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="_3">
        <item>
         <widget class="QRadioButton" name="radioPdf">
          <property name="text">
           <string>&amp;PDF version</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioHtmlzip">
          <property name="text">
           <string>&amp;HTML version (zip file)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonDownloadManual">
        <property name="text">
         <string>Down&amp;load</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
